src/sentry/static/sentry/app/components/featureBadge.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Tag from 'app/views/settings/components/tag';
+import Tooltip from 'app/components/tooltip';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import theme from 'app/utils/theme';
+import CircleIndicator from 'app/components/circleIndicator';
+
+type BadgeProps = {
+  type: 'alpha' | 'beta' | 'new';
+  variant?: 'indicator' | 'badge';
+  title?: string | null;
```

Comment:
Please why do you prefer to use null? I would only make the prop optional here

Reply:
`null` allows you to completely turn off the tooltip. I'll add a comment.

Reply:
What happens with empty string? Would a specific prop be better instead of overloading `title`?

Reply:
Yeah. I was just keeping bc since we did this one or two places. I'm good for a separate prop. fwiw I think empty string would also turn it off.